src/ray/core_worker/core_worker.cc

```diff
@@ -2340,6 +2344,9 @@ Status CoreWorker::ExecuteTask(const TaskSpecification &task_spec,
     RAY_LOG(FATAL) << "Unexpected task status type : " << status;
   }
 
+  ray::stats::STATS_tasks.Record(0, rpc::TaskStatus_Name(rpc::TaskStatus::RUNNING));
```

Comment:
Is it working if this core worker executes more than 1 task? (e.g., threaded actor or async actor)?

Reply:
Will fix... I think unifying with the other counter structure here will address.